src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1490,15 +1488,13 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
                 $container->setAlias('lock.store', new Alias('lock.'.$resourceName.'.store', false));
                 $container->setAlias('lock.factory', new Alias('lock.'.$resourceName.'.factory', false));
                 $container->setAlias('lock', new Alias('lock.'.$resourceName, false));
-                $container->setAlias(StoreInterface::class, new Alias('lock.store', false));
                 $container->setAlias(PersistingStoreInterface::class, new Alias('lock.store', false));
-                $container->setAlias(Factory::class, new Alias('lock.factory', false));
+                $container->setAlias(LockFactory::class, new Alias('lock.factory', false));
```

Comment:
line should be removed, already exists below

Reply:
done